generate-wasm-examples/generate_wasm_examples.sh

```diff
@@ -14,6 +18,26 @@ sed -i.bak 's/canvas: None,/canvas: Some("#bevy".to_string()),/' crates/bevy_win
 # setting the asset folder root to the root url of this domain
 sed -i.bak 's/asset_folder: "assets"/asset_folder: "\/assets\/examples\/"/' crates/bevy_asset/src/lib.rs
 
+export CARGO_TARGET_DIR="target"
+
+if [ "$1" = --ci ] ; then
+    # Do not optimize on --ci check,
+    # wasm-opt takes A LOT of time to run
+    wasm_opt_flag="-O0"
+else
+    echo "
+
+[profile.release]
+opt-level = \"z\"
+lto = \"fat\"
+codegen-units = 1
+" >> Cargo.toml
```

Comment:
I was wondering whether we should just add those lines to the main `Cargo.toml` in the https://github.com/bevyengine/bevy repo. I opted to add them here because it's easier :P